mappings/net/minecraft/client/render/particle/Id22RandomPopParticle.mapping

```diff
@@ -0,0 +1 @@
+CLASS em net/minecraft/client/render/particle/Id22RandomPopParticle
```

Comment:
This is a bad name but it's related to Id22 packets and the random pop sound effect